p2p/tests/integration/p2p.rs

```diff
@@ -42,6 +42,33 @@ fn setup_logger() {
     })
 }
 
+async fn try_init_network(
+    broker: &Broker,
+    public_key: &iroha_crypto::PublicKey,
+) -> Result<(String, Network<TestMessage>), iroha_p2p::Error> {
+    let address = gen_address();
+
+    time::sleep(Duration::from_millis(20)).await;
+
+    Ok((
+        address.clone(),
+        Network::<TestMessage>::new(broker.clone(), address, public_key.clone(), 100).await?,
```

Comment:
How do we know that this isn't running `clone` in a loop? Would it make sense to replace clone with a different cheaper operation?